core/crypto/whitebox/aes_whitebox.cpp-chow-sample

```diff
@@ -0,0 +1,127 @@
+// Copyright 2019 AES WBC Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Sample implementation based on https://github.com/balena/aes-whitebox, provided for the testing purpose only:
+//
+// How to use:
+//
+// 1. Rename this file to `aes_whitebox.cpp`.
+// 2. Clone https://github.com/balena/aes-whitebox repository.
+// 3. Build `aes_whitebox_compiler.cc` tool, using `make` command.
+// 4. Generate white-box tables, using `aes_whitebox_compiler aes256 {YOUR_KEY}` command.
+// 5. Copy `aes_whitebox_tables.cc` source file to this folder.
+// 6. Rebuild Godot export templates with the `whitebox_decryption=yes` option.
```

Comment:
I'm afraid that might be a bit too advanced to solve the use case of "I don't want people to read my code by simply unzipping the .pck".

What would it take to build and run the compiler ourselves in `thirdparty`, so that everything can be automated and users only need to pass `scons whitebox_decryption=yes whitebox_key={YOUR_KEY}`?